docker/README.md

```diff
@@ -37,8 +37,8 @@ docker-machine start default # Start the docker daemon
 
 # build images
 docker build -t olbase:latest -f docker/Dockerfile.olbase . # 30+ min (Win10Home/Dec 2018)
-docker build -t oldev:latest -f docker/Dockerfile.oldev .   # 10+ min (Win10Home/Dec 2018)
-docker build -t olsolr:latest -f docker/Dockerfile.olsolr . # 5+ min (Win10Home/Dec 2018)
+docker-compose build web # 10+ min (Win10Home/Dec 2018)
```

Comment:
I'm not a fan of using docker-compose as a build tool. It doesn't give us we we really want of being able to build multiple images in sequence (which is why olbase is build differently). I thought we had touched on this briefly in a community call a while ago. A shell build script or a Make task would be better and combine all the actions in a simpler to type command. Alternatively, pick and use a current docker build tool to do the job. I believe docker-compose's build functionality is deprecated too (but I can't find a reference ATM).

Reply:
Haha, I thought I just recently learned about the build option in the compose config, but it's entirely possible that I forgot. I think the docker-compose command is easier to remember? As someone who has to use these commands a lot, I got tired of having to open our docker docs every time I needed to rebuild. I think a make task would be a good idea, but it adds another layer of duplication (do we have just one command? What if I just want to build `web` (which is the only one that usually needs rebuilding)?). I think we should practice (my new favourite word:) [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) with regards to sequential builds; let's cross that bridge when we get to it :P